### PR TITLE
Hub to web

### PR DESCRIPTION
**Pull request by ikjudd**
- dashboard: bounce to homepage on disconnect
- search: fix activeView initial state for regular searches
- search: fix search result length bug
- upload: add releaseinitviahub to web
- search: fix container height
- search: in releases searchpreview should be 'artist - title'

### CODE REVIEW

File: js/hubs/components/ReleaseCreateConfirm.js [hunk 1]
```diff
@@ -115,9 +118,28 @@ const ReleaseCreateConfirm = (props) => {
...
+              return selected.join(", ");
+            }}
+            MenuProps={MenuProps}
+            inputProps={{ "aria-label": "Without label" }}
+            />
+            </FormControl>
             <Typography variant="subtitle1" mt={1} sx={{ color: "red" }}>
               ONCE PUBLISHED, YOUR RELEASE INFORMATION WILL BE PERMANENT AND YOU
-              WILL NOT BE ABLE TO EDIT IT.
+              WILL NOT BE ABLE TO EDIT IT. hahahahahahah
```
Comment (ejf89) [inline js/hubs/components/ReleaseCreateConfirm.js:142]: not sure if this is the right tone for new copy

File: js/web/components/NavSearch.js [hunk 2]
```diff
@@ -332,7 +332,8 @@ const SearchInput = styled('input')(({ theme }) => ({
   width: '15vw',
   marginRight: '20px',
   outline: 'none !important',
-  backgroundColor: '#fff',
+  //backgroundColor: '#fff',
```
Comment (ejf89) [inline js/web/components/NavSearch.js:335]: remove commented out value plz

File: js/web/components/ReleaseCreate.js [hunk 3]
```diff
@@ -322,17 +361,29 @@ const ReleaseCreate = () => {
...
-                release: info.release,
-                releaseBump: info.releaseBump,
-                releaseMint: info.releaseMint,
-                metadataUri: `https://arweave.net/${metadataResult}`,
-                release: info.release,
-                releaseBump: info.releaseBump,
-                releaseMint: info.releaseMint,
-              })
+              let result
+              console.log('selectedHub', selectedHub, info.releaseMint)
```
Comment (ejf89) [inline js/web/components/ReleaseCreate.js:365]: kill log

File: js/web/pages/[releasePubkey]/index.js [hunk 4]
```diff
@@ -48,40 +48,40 @@ const ReleasePage = (props) => {
...
-      {
-        params: {
-          releasePubkey: 'placeholder',
-        },
-      },
-    ],
-    fallback: 'blocking',
-  }
-}
+// export const getStaticPaths = async () => {
```
Comment (ejf89) [inline js/web/pages/[releasePubkey]/index.js:51]: this was commented out due to that compiling bug on friday, should be resolved - not skipped

**Review by ejf89 — CHANGES_REQUESTED** [review]
plz address comments and re request review

**Review by ejf89 — APPROVED** [review]

**Comment by ikjudd** [comment]
#458 
#466 
#468 
#460 
#469 
#470

**Pull request merged**